libraries/ui-shared/TextField/components/Label/index.jsx

```diff
@@ -15,14 +16,18 @@ const Label = (props) => {
     // eslint-disable-next-line jsx-a11y/label-has-associated-control
     <label
       htmlFor={props.name}
-      className={labelStyles}
+      className={classNames(labelStyles, props.className)}
```

Comment:
Please check. Use can concatenate class names (strings). Glamor objects will not be applied here

Reply:
removed prop, as it's not needed anymore